js_modules/dagster-ui/packages/ui-core/src/ticks/TickLogDialog.tsx

```diff
@@ -98,13 +101,61 @@ export const QueryfulTickLogsTable = ({instigationSelector, tick}: TickLogTableP
     return <TickLogsTable events={events} />;
   }
 
+  const tickStatus =
+    data?.instigationStateOrError.__typename === 'InstigationState'
+      ? data?.instigationStateOrError.tick.status
+      : undefined;
+  const instigationType =
+    data?.instigationStateOrError.__typename === 'InstigationState'
+      ? data?.instigationStateOrError.instigationType
+      : undefined;
+  const instigationLoggingDocsUrl =
+    instigationType === 'SENSOR'
+      ? 'https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#logging-in-sensors'
+      : instigationType === 'SCHEDULE'
+      ? 'https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules#logging-in-schedules'
+      : undefined;
+
   return (
     <Box
       style={{height: 500}}
       flex={{justifyContent: 'center', alignItems: 'center'}}
       padding={{vertical: 48}}
     >
-      {loading ? 'Loading logs…' : 'No logs available'}
+      {loading ? (
+        'Loading logs…'
+      ) : (
+        <NonIdealState
+          icon="no-results"
+          title="No logs to display"
+          description={
+            <Box flex={{direction: 'column'}}>
```

Comment:
You can use `gap: 12` here instead of the padding on the individual body elements.